src/App.tsx

```diff
@@ -15,48 +17,123 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Idle = 'idle',
+  Alphabet = 'alphabet',
+  Length = 'length',
+}
+
+interface Filter {
+  active: SortType;
```

Comment:
We have more than one filter, so it is necessary to specify that we will store exactly the type of filter
```suggestion
  sortType: SortType;
```